src/services/assetService.ts

```diff
@@ -146,7 +147,12 @@ export class AssetService {
 
     // If extension of a file was spoofed, we fetch only first 4 bytes of the file and read MIME type
     public static async getMimeType(uri: string): Promise<string[]> {
-        const first4bytes: Response = await fetch(uri, { headers: { range: `bytes=0-${mimeBytesNeeded}` } });
+        let first4bytes: Response;
+        try {
+            first4bytes = await fetch(uri, {headers: {range: `bytes=0-${mimeBytesNeeded}`}});
+        } catch (e) {
+            throw new Error(e);
```

Comment:
this will still show the error to the user, even though it's unblocking and the user will not know what to do with the "failed to fetch" message

Reply:
So it doesn't change the current user experience at all